matrix-sdk-android/src/main/res/values/strings.xml

```diff
@@ -139,12 +138,6 @@
     <string name="medium_email">Email address</string>
     <string name="medium_phone_number">Phone number</string>
 
-    <!-- Reply to -->
-    <string name="reply_to_an_image">sent an image.</string>
-    <string name="reply_to_a_video">sent a video.</string>
-    <string name="reply_to_an_audio_file">sent an audio file.</string>
-    <string name="reply_to_a_file">sent a file.</string>
```

Comment:
Should be also removed from the translations file. I'm surprised that lint does not complain...

Reply:
But maybe for the moment we should keep the strings, even if they are not used anymore.

Reply:
I changed the commits so all the strings are removed in a single commit that can be easily reverted if needed